src/main/scala/org/ergoplatform/http/api/TransactionsApiRoute.scala

```diff
@@ -48,32 +47,25 @@ case class TransactionsApiRoute(readersHolder: ActorRef, nodeViewActorRef: Actor
       _.fold(
         e => BadRequest(s"Malformed transaction: ${e.getMessage}"),
         _ => {
-          nodeViewActorRef ! LocallyGeneratedTransaction[ErgoTransaction](tx)
+          processFn(tx)
           ApiResponse(tx.id)
         }
       )
     }
   }
 
-  def getUnconfirmedTransactionsR: Route = (path("unconfirmed") & get & paging) { (offset, limit) =>
-    ApiResponse(getUnconfirmedTransactions(offset, limit))
+  def sendTransactionR: Route = (post & entity(as[ErgoTransaction])) { tx =>
```

Comment:
I would suggest adding `pathEnd` to avoid possible confusion.

Reply:
Added!